edx_proctoring/api.py

```diff
@@ -1235,7 +1235,9 @@ def _check_eligibility_of_enrollment_mode(credit_state):
     # Also make an exception for the honor students to take the "practice exam" as a proctored exam.
     # For the rest of the enrollment modes, None is returned which shows the exam content
     # to the student rather than the proctoring prompt.
-    return credit_state['enrollment_mode'] == 'verified'
+    if credit_state:
+        credit_state['enrollment_mode'] == 'verified'
+    return credit_state
```

Comment:
I think it is better to explicitly return `False` here. Caller of this function would be expecting a boolean. Some caller of this function may use this function like
``` python 
if _check_eligibility_of_enrollment_mode == True/False
```